packages/e2e-tests/src/tests/PKPEthers.spec.ts

```diff
@@ -0,0 +1,861 @@
+          { name: 'to', type: 'Person[]' },
+          { name: 'contents', type: 'string' },
+        ],
+        Person: [
+          { name: 'name', type: 'string' },
+          { name: 'wallets', type: 'address[]' },
+        ],
+      },
+    };
+
+    const signature = await ethRequestHandler({
+      signer: pkpEthersWallet,
+      payload: {
+        method: 'eth_signTypedData_v4',
+        params: [alice.pkp?.ethAddress, JSON.stringify(msgParams)],
+      },
+    });
+
+    const recoveredAddr = recoverTypedSignature({
+      data: msgParams as any,
+      signature: signature,
+      version: SignTypedDataVersion.V4,
+    });
+
+    expect(signature.length).toEqual(132);
+
+    expect(recoveredAddr.toLowerCase()).toEqual(
+      alice.pkp?.ethAddress.toLowerCase()
+    );
+  } catch (e) {
+    throw new Error(`❌ ${(e as Error).toString()}`);
+  } finally {
+    devEnv.releasePrivateKeyFromUser(alice);
+  }
+};
+
+const signWithAuthContext = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  const alice = await devEnv.createRandomPerson();
+
+  const pkpEthersWallet = new PKPEthersWallet({
+    pkpPubKey: alice.pkp?.publicKey!,
+    litNodeClient: devEnv.litNodeClient!,
+    authContext: {
+      getSessionSigsProps: {
+        authNeededCallback: async function (
+          params: AuthCallbackParams
+        ): Promise<AuthSig> {
+          const toSign = await createSiweMessageWithRecaps({
+            uri: params.uri!,
+            expiration: params.expiration!,
+            resources: params.resourceAbilityRequests!,
+            walletAddress: alice.wallet.address,
+            nonce: await devEnv.litNodeClient?.getLatestBlockhash()!,
+            litNodeClient: devEnv.litNodeClient,
+          });
+
+          const authSig = await generateAuthSig({
+            signer: alice.wallet,
+            toSign,
+          });
+
+          return authSig;
+        },
+        resourceAbilityRequests: [
+          {
+            resource: new LitPKPResource('*'),
+            ability: LitAbility.PKPSigning,
+          },
+          {
+            resource: new LitActionResource('*'),
+            ability: LitAbility.LitActionExecution,
+          },
+        ],
+      },
+    },
+  });
+};
+
+const ethPersonalSign = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  const alice = await devEnv.createRandomPerson();
+  const pkpSessionSigs = await generator(devEnv, alice);
+
+  const pkpEthersWallet = new PKPEthersWallet({
+    litNodeClient: devEnv.litNodeClient!,
+    pkpPubKey: alice.pkp?.publicKey!,
+    controllerSessionSigs: pkpSessionSigs,
+  });
+
+  await pkpEthersWallet.init();
```

Comment:
If that is the case wouldn't we not want to catch that error and let the test runner report the error? Since we would just have to wait for a new key and restart the test essentially?